src/main/java/parser/Parser.java

```diff
@@ -0,0 +1,73 @@
+package parser;
+
+import commands.ByeCommand;
+import commands.Command;
+import commands.DeadlineCommand;
+import commands.DeleteCommand;
+import commands.EventCommand;
+import commands.ListCommand;
+import commands.MarkCommand;
+import commands.ToDoCommand;
+import exceptions.DukeException;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+public class Parser {
+
+    public static Command parse(String input) throws DukeException {
+        String command, description;
+
+        String[] words = input.split(" ", 2);
+        command = words[0];
+        if (words.length > 1) description = words[1];
```

Comment:
Body of conditionals should still be on a separate line even for a single line body